adm/style/ppde_transactions.html

```diff
@@ -1,11 +1,112 @@
 {% INCLUDE 'overall_header.html' %}
 
+<a href="{{ U_BACK }}" style="float: {{ S_CONTENT_FLOW_END }};">&laquo; {{ lang('BACK') }}</a>
+
+{% if S_ADD %}
+
+<h1>{{ lang('PPDE_MT_TITLE') }}</h1>
+<p>{{ lang('PPDE_MT_TITLE_EXPLAIN') }}</p>
+<p><strong>{{ lang('PPDE_MT_REQUIRED_CHARACTER') }}</strong> {{ lang('PPDE_MT_REQUIRED_EXPLAIN') }}</p>
+
+{% if S_ERROR %}
+<div class="errorbox">
+	<h3>{{ lang('WARNING') }}</h3>
+	<p>{{ ERROR_MSG }}</p>
+</div>
+{% endif %}
+
+<form id="manual_transaction" method="post" action="{{ U_ACTION }}">
+	<fieldset>
+		<legend>{{ lang('PPDE_MT_DETAILS') }}</legend>
+		<dl>
+			<dt><label>{{ lang('PPDE_MT_USERNAME') ~ lang('PPDE_MT_REQUIRED_CHARACTER') ~ lang('COLON') }}</label><br><span>{{ lang('PPDE_MT_USERNAME_EXPLAIN') }}</span></dt>
```

Comment:
Could you check if this is all resolved?

Reply:
Adding the attribute `for` is not enough.
You must add the attribute `id` to the form controls which need to get the focus when you click on the label.
You can take a look on the template `ppde_settings.html` or all phpBB Template.

Reply:
I think all input elements already have an ID. Some elements don't have a label associated with them because there is more than one element for the label:

* The `#mc_currency` select element. The label is already associated with the `#mc_gross` input element.
The `#payment_date_month` and `#payment_date_day` elements. The label is already associated with the `#payment_date_year` element.

Let me know if I missed anything. :grin:

Reply:
My bad... I didn't see the `id` and I had a problem with the browser cache.
Issue solved 👍